ui/src/components/overview/TournamanentCard.tsx

```
@@ -266,171 +255,30 @@ export const TournamentCard = ({
+          <div className="flex flex-row items-center gap-1.5 min-w-0 flex-1">
+            <Badge
+              variant={tournamentStatus.variant}
+              className="text-[10px] sm:text-xs px-1.5 py-0 sm:py-0.5 rounded-md h-5 flex-shrink-0"
+            >
+              {tournamentStatus.text}
+            </Badge>
+            {isRestricted && (
+              <Lock className="w-3 h-3 text-brand-muted flex-shrink-0" />
+            )}
+            <p className="truncate min-w-0 font-brand text-sm sm:text-base">
               {feltToString(tournament?.metadata?.name!)}
```

Biome (2.3.13)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)